### PR TITLE
Added support for paste.

### PR DESCRIPTION
**Pull request by dudeNumber4**
I just can't figure out how to debug my command.  I don't know how/where `registerCommands` is supposed to get called.  Reference [feature request](https://github.com/alefragnani/vscode-copy-word/issues/40).  Disregard - I had never run `vscode:prepublish`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "copy-word",
-  "displayName": "Copy Word in Cursor",
-  "description": "Copy/Cut the current word when there is no selection",
-  "version": "3.6.0",
+  "displayName": "Copy/Paste Word in Cursor",
+  "description": "Cut/Copy/Paste the current word when there is no selection",
+  "version": "3.7.0",
```
Comment (alefragnani) [inline package.json:5]: Undo these changes. I'm not sure the new `Paste` command needs to update the extension's name/description

File: package.json [hunk 2]
```diff
@@ -72,17 +77,17 @@
...
     "@typescript-eslint/eslint-plugin": "^5.1.0",
     "@typescript-eslint/parser": "^5.1.0",
+    "@vscode/test-electron": "^1.6.2",
     "esbuild": "^0.13.12",
     "eslint": "^8.1.0",
     "eslint-config-vscode-ext": "^1.1.0",
     "mocha": "^9.1.3",
-    "typescript": "^4.4.4",
-    "@vscode/test-electron": "^1.6.2"
+    "typescript": "^4.6.2"
```
Comment (alefragnani) [inline package.json:91]: Undo this change. There is no need to update TS.

Comment (alefragnani) [inline package.json:91]: Undo this change. There is no need to update TS.

File: README.md [hunk 3]
```diff
@@ -8,12 +8,9 @@
   <a title="Learn more about Copy Word in Cursor" href="http://github.com/alefragnani/vscode-copy-word"><img src="https://raw.githubusercontent.com/alefragnani/vscode-copy-word/master/images/vscode-copy-word-logo-readme.png" alt="Copy Word in Cursor Logo" width="70%" /></a>
 </p>
 
-# What's new in Copy Word in Cursor 3.6
+# What's new in Copy Word in Cursor 3.7
```
Comment (alefragnani) [inline README.md:11]: Thanks, but undo file changes. Whenever I update the extension, I like to review its content.

Comment (alefragnani) [inline README.md:11]: Thanks, but undo file changes. Whenever I update the extension, I like to review its content.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
+## [3.7.0] - 2019-11-22
+### Added
+- Support **Paste**
+
```
Comment (alefragnani) [inline CHANGELOG.md:4]: Thanks, but undo file changes. Whenever I update the extension, I like to review its content.

Comment (alefragnani) [inline CHANGELOG.md:4]: Thanks, but undo file changes. Whenever I update the extension, I like to review its content.

**Review by alefragnani — CHANGES_REQUESTED** [review]
Please, update the PR with the provided review. 

Once updated, I'll try out.

Thank you

**Comment by alefragnani** [comment]
Hi @dudeNumber4 ,

First of all, thank you for your PR. I made a few comments, and would like you to take a look. Those 3 files `README.md`, `CHANGELOG.md` and `package-lock.json` could be reverted (removed from PR) and only package.json needs update. The extension's code itself, seems to be ok.

About debugging, I've noticed this issue after updating the extension to use **esbulid**. It requires you to be _built_ (`npm run build`) every time you update the code, because there it has no `watch` feature. I'm reverting this change in #43 , because web extension samples uses **WebPack** for easier packaging. With **WebPack**, which scripts supports watch, whenever you change the code, the debug works as expected.

I can merge #43 into main, so you could update your branch and retry with WebPack. Otherwise, simply follow these esbuild instructions to properly debug the extension. What do you prefer?

Thank you

**Comment by dudeNumber4** [comment]
I was able to debug; it should be good to merge.

**Comment by dudeNumber4** [comment]
Is there anything else blocking this from being merged?

**Comment by alefragnani** [comment]
Not at all. 

Just my free time to test/merge 😉

**Comment by alefragnani** [comment]
My plans are for this weekend

**Comment by alefragnani** [comment]
Just published v3.7.0 with your PR.

This new release also works on the **web** (github.dev and vscode.dev)

Thank you

**Pull request merged**